core/src/test/scala/pureconfig/DurationConvertSuite.scala

```diff
@@ -82,4 +82,7 @@ class DurationConvertSuite extends FlatSpec with Matchers with EitherValues {
         ex.head shouldBe a[CannotConvert]
     }
   }
+  it should "fromS(fromD(Duration.Undefined)) == Duration.Undefined" in {
```

Comment:
Oh, sure. I agree that's better I was just copying the test that was provided in #184.